src/Extensions/Logging/Internal/EventSource/OmexLogEventSource.cs

```diff
@@ -29,9 +29,10 @@ internal sealed class OmexLogEventSource : EventSource
 			string tagId,
 			string tagName,
 			int threadId,
-			string message) =>
+			string message,
+			bool isHealthCheck) =>
 			WriteEvent((int)EventSourcesEventIds.LogError, applicationName, serviceName, agentName, buildVersion, processName, partitionId, replicaId,
-				activityId, activityTraceId, correlationId, transactionId, level, category, tagId, tagName, threadId, message);
+				activityId, activityTraceId, correlationId, transactionId, level, category, tagId, tagName, threadId, message, isHealthCheck);
 
 		[Event((int)EventSourcesEventIds.LogWarning, Level = EventLevel.Warning, Message = "{12}:{13} {16}", Version = 6)]
```

Comment:
It's probably a good idea to bump `Version` of all events

Reply:
Have incremented all events to 7